Installation/Source/buren.openpdc.source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "title": "OpenPDC API",
+    "$id": "https://buren.nl/source/buren.openpdc.source.json",
+    "$schema": "https://docs.commongateway.nl/schemas/Gateway.schema.json",
+    "description": "Yard's OpenPDC SDG API",
+    "version": "0.0.1",
+    "location": "!ChangeMe!",
```

Comment:
It is oke (and maybe better) to add locations to sources in /installation/source.json files. Just don't add auth info

Reply:
That depends, in some cases the source location might vary depending on the environment or customer. Also, it is perfectly fine by me to add source locations that are obviously public, but for locations that might be secret it is better to not save it in your repo.

Reply:
For public (test) apis or test apis with auth it shouldn't be a problem.

A few plus points for adding location in the .json file:
Adding location here prevents overriding it when versions ever get updated for this source.
Also makes it easier to find the correct auth in passwork (a long time later from now)